docs/topics/tour/kotlin-tour-collections.md

```diff
@@ -230,14 +234,24 @@ fun main() {
 
 ## Map
 
+Maps store items as key-value pairs. You access the value by referencing the key. You can imagine a map like a food menu.
+You can find the price (value), by finding the food (key) you want to eat. Maps are useful if you want to look up a value
+without using a numbered index, like in a list.
+
+> * Every key in a map must be unique so that Kotlin can understand which value you want when you try to access it. 
```

Comment:
I'm a bit confused with **try** here.
"Kotlin can understand which value you want to get" or "which value you want to get when you access it"?

What do you think?

Reply:
I like your first suggestion best 👍